clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -599,9 +599,10 @@ Expected<StringRef> writeOffloadFile(const OffloadFile &File) {
 
   StringRef Prefix =
       sys::path::stem(Binary.getMemoryBufferRef().getBufferIdentifier());
-
-  auto TempFileOrErr = createOutputFile(
-      Prefix + "-" + Binary.getTriple() + "-" + Binary.getArch(), "o");
+  std::string Filename =
+      (Prefix + "-" + Binary.getTriple() + "-" + Binary.getArch()).str();
+  llvm::replace(Filename, ':', '-');
```

Comment:
```suggestion
  llvm::replace(Filename, ':', '_');
```
inconsistency with PR description

Reply:
Accidentally held shift, thanks.